portfolio/src/main/webapp/script.js

```diff
@@ -58,25 +58,25 @@ function getComments() {
     // reference its fields to create HTML content
 
     const commentContainer = document.getElementById('comments-container');
-    addCommentElements(commentContainer, comments);
+    addCommentElements(commentContainer, comments, false);
   });
 }
 
 /** Creates a <div> element containing comment. */
-function addCommentElements(commentContainer, comments) {
+function addCommentElements(commentContainer, comments, isReply) {
```

Comment:
This code should be sufficient here - 

for (var i = 0; i < comments.length; i++) {      
    commentContainer.appendChild(createCommentElement(comments[i], isReply));
}
return commentContainer;